recipes/r-pythoninr/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pythoninr
+  version: "0.1_4"
+
+source:
+  fn: PythonInR_0.1-4.tar.gz
+  url: https://cran.r-project.org/src/contrib/PythonInR_0.1-4.tar.gz
+  sha256: ee9d5c33580b9273d80e43ed4d83dae81de2fc35cc92e310feefe5e4c12c394f 
+
+build:
+  number: 0
+  skip: true  # [py34]
+  skip: true  # [win32]
```

Comment:
If you do end up deciding to skip for multiple reasons, you can put these on the same line with

```
  skip: true  # [py34 or win32]
```